meson.build

```diff
@@ -39,7 +39,7 @@ endif
 
 deps_for_imv = [
   dependency('pangocairo'),
-  dependency('gl'),
+  dependency('opengl'),
```

Comment:
I wonder if meson has some module for OpenGL linking? CMake supports selecting if you want `libglvnd` or "legacy" behavior/

Reply:
Looked at it again, I think the best way to fix it would via meson as `dependency('gl')` is actually supposed to manage this kind of thing. (see https://github.com/mesonbuild/meson/blob/adfcf77109a2c73d88295a0fc60555b34da3eebe/mesonbuild/dependencies/ui.py#L41 ) But I'm not sure how to fix it and it doesn't seems much likely to be fixed relatively soon ( https://github.com/mesonbuild/meson/issues/3679 ).

I'll try other ways of doing it like with `dep.found()`…